### PR TITLE
For Beginners document and reference in read me added

### PR DESCRIPTION
**Pull request by rtilla1**
I added a relative link and reference to the "for beginners" documentation in the read.me file. I also added a step-by-step walk-through for turning any old machine into a CLAW environment.

### CODE REVIEW

File: forBeginners.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+but if it does, that's not a problem. It should be a machine that has been totally wiped or only has files that can be deleted. If you have any files on it that you would like to save, you should back them up elsewhere. 
+
+  Other requirements this computer will (ideally) meet:
+  1. 3GB of RAM
+  2. 4GB of hard drive memory free (25 GB required for Ubuntu desktop)?
+  3. Internet connection
+  
+  >If you're not sure how much RAM and free memory on your hard drive is on the computer and you have a Windows computer, you can follow this [tutorial](https://www.computerhope.com/issues/ch000149.htm).
+
+  >If you don't have an operating system running, you can try to install the Ubuntu operating system below and follow these [steps](**return here to include link**)
```
Comment (whikloj) [inline forBeginners.md]: I think you forgot to include a link 😉

**Comment by rtilla1** [comment]
Sorry...I had to do quite a bit of research and editing to correct this missing link. I believe it's corrected now.

**Comment by whikloj** [comment]
I haven't actually run through this process, but it all looks correct and if we find some corrections later we can handle them then.
👍

**Comment by whikloj** [comment]
@rtilla1 this is good, but I couldn't find your name (or Tulane) on the list of current CLAs.
https://github.com/Islandora/islandora/wiki/Contributor-License-Agreements

Have you signed an individual CLA or has Tulane submitted one? We'll need one before I can merge this, my apologizes I should have caught this earlier.

**Comment by rtilla1** [comment]
I have not signed a personal one. I’m surprised Tulane hasn’t submitted one. I’ll check with Jeff Rubin because we definitely should sign one as an institution.

Rachel Tillay
Pronouns: they/them/theirs
Cataloging and Metadata Librarian
Tulane University
rtillay@tulane.edu<mailto:rtillay@tulane.edu>
T: 504-247-1561

**Comment by whikloj** [comment]
@manez @dannylamb what do we do here? 

We had a similar situation in Fedora where someone had come and helped update a bunch of dependency versions (so no code changes) and disappeared. We merged it with the plan to add a note to the CLA table indicating this deviance from the norm.

Or we dump it 🤷‍♂️

**Comment by manez** [comment]
@rtilla1 is certainly still around, but might not be getting her GitHub notices. I'll email her off-thread and see if she's made any CLA progress.

**Comment by manez** [comment]
And it's now official! We have a cCLA from Tulane and @rtilla1 's awesome work is good to go 🎉 🎉 🎉

**Pull request merged**